mitmproxy-rs/Cargo.toml

```diff
@@ -29,7 +29,7 @@ tokio = { version = "1.41", features = ["macros", "net", "rt-multi-thread", "syn
 boringtun = "0.6"
 tar = "0.4.43"
 console-subscriber = { version = "0.4.1", optional = true }
-
+nix = "0.17.0"
```

Comment:
Is there a reason why you added such an old version of `nix` here? The 0.17 release is from 2020, the current version is 0.29.

Reply:
oops, thanks for spotting it out